joint_trajectory_controller/include/joint_trajectory_controller/init_joint_trajectory.h

```diff
@@ -308,10 +330,13 @@ Trajectory initJointTrajectory(const trajectory_msgs::JointTrajectory&       msg
     if (msg_it == msg.points.end())
     {
       ros::Duration last_point_dur = time - (msg_start_time + (--msg_it)->time_from_start);
-      ROS_WARN_STREAM("Dropping all " << msg.points.size() <<
-                      " trajectory point(s), as they occur before the current time.\n" <<
-                      "Last point is " << std::fixed << std::setprecision(3) << last_point_dur.toSec() <<
-                      "s in the past.");
+      error_string = "Dropping all " + std::to_string(msg.points.size());
+      error_string += " trajectory point(s), as they occur before the current time.\n";
+      error_string += "Last point is " + std::to_string(last_point_dur.toSec());
+      error_string += "s in the past.";
+      ROS_WARN_STREAM(error_string);
+      if (options.error_string)
```

Comment:
Could we please use full brackets for these to match the format of the rest of the repo?

```
if (options.error_string)
{
  *options.error_string = error_string;
}
```

Reply:
Other than this it's all green lights from me too